messages/recordcounts.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commandDescription": "print record counts for the given objects",
+  "examples": [
+    "sfdx force:limits:recordcounts:display -o Account,Contact,Lead,Opportunity",
+    "sfdx force:limits:recordcounts:display -o Account,Contact -u me@my.org"
+  ],
+  "objectnamesFlagDescription": "comma-separated list of object names for which to print record counts"
```

Comment:
Small edit:  "comma-separated list of standard or custom object names for which to display record counts"

As with other comment, if this works only for standard objects then update as needed.

Reply:
Incorporated/changed to "comma-separated list of API names of standard or custom objects for which to display record counts".